v2/googlecloud-to-neo4j/src/main/java/com/google/cloud/teleport/v2/neo4j/model/helpers/JobSpecMapper.java

```diff
@@ -38,19 +39,45 @@ public class JobSpecMapper {
   private static final Logger LOG = LoggerFactory.getLogger(JobSpecMapper.class);
 
   public static ImportSpecification parse(String jobSpecUri, OptionsParams options) {
-    var json = fetchContent(jobSpecUri);
-    var spec = new JSONObject(json);
+    String content = fetchContent(jobSpecUri);
+    JSONObject spec = getJsonObject(content);
+
     if (!spec.has("version")) {
       return parseLegacyJobSpec(options, spec);
     }
+
     try {
       // TODO: interpolate runtime tokens into new spec elements
-      return ImportSpecificationDeserializer.deserialize(new StringReader(json));
+      return ImportSpecificationDeserializer.deserialize(new StringReader(content));
     } catch (SpecificationException e) {
       throw validationFailure(e);
     }
   }
 
+  private static JSONObject getJsonObject(String content) {
+    JSONObject spec;
+
+    try {
+      spec = new JSONObject(content);
```

Comment:
nit: I'd rather use `return` directly and in the `catch` block than rely on a variable